docs/creating-components.md

```diff
@@ -156,67 +156,56 @@ const CountableDiv = Countable('div');
 const CountableDivAsDataNumber = CountableAsDataNumber('div');
 ```
 
-## Adding Design-Time Customizations to Components
+## Customizing Components at Design-Time
 
-> NOTE: `factory()` is deprecated and will be removed in 2.0
+> NOTE: `factory()`, the previous experimental technique for doing this, is deprecated and will be removed in 2.0
 
-While many of our components are easily configured using properties, we've found that sometimes an app
-needs the more flexibility. Generally, this type of customization isn't variable at
-run-time and is primarily used to adjust the visual design of a component. To support this use case,
-we've introduced the `factory()` function.
+Occasionally, you'll want to modify the appearance of an Enact component, and the majority of that time, you can simply apply external styling to the outer-most element of the component, via `className` or `style`. However, what if you need to customize one of the deeper child elements?
 
-Like `hoc()`, `factory()` accepts a configuration object which is merged with the component's default
-configuration. Unlike `hoc()`, `factory()` only supports a known set of configuration keys rather
-than an arbitrary set. Currently, the only supported key is `css`, which allows authors to provide
-a custom CSS class name map which is merged with the component's CSS class name map. This feature
-allows for overriding the style of particular children within the component's internal
-hierarchy.
+We've got you covered! In Enact 2.0 we've added a built-in theming capability to make this significantly easier and even safer. Now, components built with the `kind` feature can use the `publicClassNames` key in their `styles` block of their component definition.
```

Comment:
This part here seems to imply that `publicClassNames` is a feature that an end developer uses.  Can you make this more clear that it's the framework (or customizable component author) that uses this and the developer only has to pass in the `css` prop?  Perhaps save mentioning `publicClassNames`?

Reply:
I did sorta switch gears there, didn't I? Yeah, I'll rework this and maybe move it to another section or rewrite it so it's just referring to the public-facing API, rather than the component setup aspect.